README.md

```diff
@@ -20,6 +20,15 @@ Well, with `what` all you have to do is ask `what "5f4dcc3b5aa765d61d8327deb882c
 
 `what`'s job is to **identify _what_ something is.** Whether it be a file or text! Or even the hex of a file! What about text _within_ files? We have that too! `what` is recursive, it will identify **everything** in text and more! 
 
+
+# :question: Installation 
+#### Windows
+ `pip install pyWhat`
+#### Linux | Darwin
+ `pip3 install pyWhat`
+ 
+#### Cloning from Source
+`git clone https://github.com/bee-san/pyWhat`
```

Comment:
What do you think about including instructions here on how to build from source? 💖